src/index.js

```diff
@@ -91,6 +95,16 @@ export default class AutoCheckElement extends HTMLElement {
   }
 }
 
+function setLoadingState(event: Event) {
+  const input = event.currentTarget
+  if (!(input instanceof HTMLInputElement)) return
+
+  const autoCheckElement = input.closest('auto-check')
+  if (autoCheckElement instanceof AutoCheckElement && autoCheckElement.required) {
+    input.setCustomValidity('Verifying…')
```

Comment:
`auto-check-send` should happen here and use the custom validation message.

https://github.com/github/auto-check-element/blob/954cfd7dec1f8a3682a27c39c4cdd1a1bcf6736a/src/index.js#L155-L174

Reply:
I think we might need another event since `auto-check-send` is doing two things. It's letting the application overwrite the default loading validity message but also supplying it with the form data of the request it's about to send so that the application can change it, if it wants.

Reply:
I pushed changes that do ☝️, let me know what you think.